eth/precompiles/blake2.py

```diff
@@ -11,6 +10,11 @@
     BaseComputation,
 )
 
+try:
+    from blake2b import compress as blake2b_compress
+except ModuleNotFoundError:
```

Comment:
Just noting that I've used `ImportError` for this kind of pattern in the past for which `ModuleNotFoundError` is a child of. The difference is that `ModuleNotFoundError` will only catch if the module can not be found whereas `ImportError` will additionally catch cases where the module might be found but can not be imported nonetheless. But I'm not sure if there's a clear recommendation which one to use for cases like this. Just pointing it out in case you have a preference.